File: Login/back/src/main/java/org/fgjcdmx/DTO/UsuarioDto.java
package org.fgjcdmx.DTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class UsuarioDto {
    private Long numEmpl;
    private String password;
    private String passwordRepeat;
    private Boolean validarPass;
    private String nombres;
    private String apellidoP;
    private String apellidoM;
    private Integer area;
    private Integer puesto;
    private Integer rol;
    private Integer status;
    private Date fecha;
}

File: Login/front/src/models/DTO/UsuarioDto.ts
import { Area } from "../Types/AreaType";
import { Generico } from "../Types/GenericoType";
import { Puesto } from "../Types/PuestoType";

export interface UsuarioDTO {
    numEmpl: number;
    nombres: string;
    apellidoP: string;
    apellidoM: string;
    password: string;
    passwordRep: string;
    validarPass: boolean;
    area: Area;
    rol: Generico;
    puesto: Puesto;
    fecha: Date;
    status: number;
}